addons/sourcemod/scripting/AFKManager.sp

```diff
@@ -512,26 +533,44 @@ public int Native_GetClientIdleTime(Handle plugin, int numParams)
 {
 	int client = GetNativeCell(1);
 
-	if(client > MaxClients || client <= 0)
+	if (client > MaxClients || client <= 0)
 	{
 		ThrowNativeError(SP_ERROR_NATIVE, "Client is not valid.");
 		return -1;
 	}
 
-	if(!IsClientInGame(client))
+	if (!IsClientInGame(client))
 	{
 		ThrowNativeError(SP_ERROR_NATIVE, "Client is not in-game.");
 		return -1;
 	}
 
-	if(IsFakeClient(client))
+	if (IsFakeClient(client))
 	{
 		ThrowNativeError(SP_ERROR_NATIVE, "Client is fake-client.");
 		return -1;
 	}
 
-	if(!g_Players_bEnabled[client])
+	if (!g_Players_bEnabled[client])
 		return 0;
 
 	return GetTime() - g_Players_iLastAction[client];
 }
+
+#if defined _EventsManager_included
+public void Events_OnEventPreStarted()
+{
+	g_bEventLoaded = true;
+}
+
+public void Events_OnPromotingAdmins()
+{
+	// Verify Admins who get promoted to Event Manager
+	CheckEveryoneAdminImmunity();
+}
+
+public void Events_OnEventStopped()
```

Comment:
Event plugin might be unloaded during an event

Reply:
This forward is also fired OnPluginEnd and OnPluginPauseChange = true

Reply:
oh okay nice